charts/lagoon-remote/Chart.yaml

```diff
@@ -49,4 +49,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: Update ssh-portal to latest version.
+      description: Update storage-calculator to the latest version
```

Comment:
can we call out the version we're updating to - otherwise the changelog will just say latest :-)